trt-llm-engine-builder-templates/llama-3_1-70b-instruct/low_ttft/README.md

```diff
@@ -14,14 +14,14 @@ This deployment is tailored for applications that require rapid response times,
 
 The template uses the following key configuration parameters:
 
-| Property             | Value  | Description |
-|----------------------|--------|-------------|
-| GPU                  | 2xH100 | Two NVIDIA H100 GPUs |
-| `max_batch_size`     |   4    | Allows processing up to 4 requests simultaneously |
-| `quantization_type`  |`fp8_kv`| FP8 quantization for key and value tensors, balancing performance and accuracy |
-| `max_input_len`      |  4096  | Maximum number of input tokens |
-| `max_output_len`     |  1024  | Maximum number of output tokens |
-| `prefix_caching`     | `true` | Enables caching of prefix computations for faster responses |
+| Property            | Value    | Description                                                                    |
+| ------------------- | -------- | ------------------------------------------------------------------------------ |
+| GPU                 | 2xH100   | Two NVIDIA H100 GPUs                                                           |
+| `max_batch_size`    | 4        | Allows processing up to 4 requests simultaneously                              |
+| `quantization_type` | `fp8_kv` | FP8 quantization for key and value tensors, balancing performance and accuracy |
+| `max_input_len`     | 4096     | Maximum number of input tokens                                                 |
+| `max_output_len`    | 1024     | Maximum number of output tokens                                                |
+| `prefix_caching`    | `true`   | Enables caching of prefix computations for faster responses                    |
```

Comment:
let's link to some blog about this  that would describe it better

"prefix computation" sounds like filler words to me. I think what we should say something like "Reuse the KV cache across requests, improving performance when different requests share a common start to the prompt"